docs/operations/multi-tenancy.md

```diff
@@ -0,0 +1,13 @@
+# Loki Multi-Tenancy
+
+Loki is a multitenant system; requests and data for tenant A are isolated from
+tenant B. Requests to the Loki API should include an HTTP header
+(`X-Scope-OrgID`) that identifies the tenant for the request.
+
+Tenant IDs can be any alphanumeric string; limiting them to 20 bytes is
```

Comment:
Ah, this is difficult because Loki doesn't do any limiting on the header size (other than the HTTP header limit which is 1MB for Go). I'll try to rephrase to make this clearer.